ddtrace/tracer/civisibility_writer_test.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package tracer
+
+import (
+	"errors"
+	"fmt"
+	"github.com/tinylib/msgp/msgp"
```

Comment:
this will probably fail on lint check at some point (import order)

Reply:
fixed imports